tests/arith_ext/arith-ext-to-arith.mlir

```diff
@@ -1,4 +1,229 @@
-// RUN: heir-opt -arith-ext-to-arith --split-input-file %s | FileCheck %s
+// RUN: heir-opt -arith-ext-to-arith --split-input-file %s | FileCheck %s --enable-var-scope
+
+// CHECK-LABEL: @test_lower_simple_add
+// CHECK-SAME: (%[[LHS:.*]]: [[TYPE:.*]], %[[RHS:.*]]: [[TYPE]]) -> [[TYPE]] {
+func.func @test_lower_simple_add(%lhs : i8, %rhs : i8) -> i8 {
+  // CHECK-NOT: arith_ext.add
+  // CHECK: %[[ADD:.*]] = arith.addi %[[LHS]], %[[RHS]] : [[TYPE]]
+  // CHECK: %[[CMOD:.*]] = arith.constant 17 : [[TYPE]]
+  // CHECK: %[[REM:.*]] = arith.remui %[[ADD]], %[[CMOD]] : [[TYPE]]
+  // CHECK: return %[[REM]] : [[TYPE]]
+  %res = arith_ext.add %lhs, %rhs {modulus = 17 }: i8
```

Comment:
None of these moduli specify a type. You should test it with `modulus = 17 : i128` or similar to make sure that works.

Reply:
Nevermind, it's an untyped attribute.